### PR TITLE
add a footer prompt for ET ratings

### PR DESCRIPTION
**Pull request by zbtirrell**
We also changed the CTA slightly.

See: https://central.tri.be/issues/63557

### CODE REVIEW

**Comment by tr1b0t** [comment]
:bangbang: This Pull Request is against a Central ticket for "NR Maintenance Release". It shouldn't be merged until the Central ticket has been changed to "Current Release" or "CR Maintenance Release".

File: src/Tribe/Credits.php [hunk 1]
```diff
@@ -40,15 +45,27 @@ public function rating_nudge( $footer_text ) {
...
+			if ( class_exists( 'Tribe__Events__Main' ) ) {
+				$review_url = 'https://wordpress.org/support/view/plugin-reviews/the-events-calendar?filter=5';
+
+				$footer_text = sprintf(
+					esc_html__( 'Rate %1$sThe Events Calendar%2$s %3$s', 'tribe-common' ),
+					'<strong>',
+					'</strong>',
+					'<a href="' . $review_url . '" target="_blank">&#9733;&#9733;&#9733;&#9733;&#9733;</a>'
+				);
+			}
```
Comment (bordoni) [inline src/Tribe/Credits.php]: can we use `} else {`?

File: src/Tribe/Credits.php [hunk 2]
```diff
@@ -23,7 +23,12 @@ public function hook() {
 	 * @return void
 	 **/
 	public function html_comment_credit( $after_html ) {
-		$html_credit = "\n<!--\n" . esc_html__( 'This calendar is powered by The Events Calendar.', 'tribe-common' ) . "\nhttp://m.tri.be/18wn\n-->\n";
+
+		if ( ! class_exists( 'Tribe__Events__Main' ) ) {
+			return;
```
Comment (bordoni) [inline src/Tribe/Credits.php]: return `$after_html`, to prevent breaking if anything was applied to that filter before.

**Comment by bordoni** [comment]
👍

**Pull request merged**